src/hb-directwrite-shape.cc

```diff
@@ -65,14 +65,49 @@ struct hb_directwrite_font_data_t {};
 hb_directwrite_font_data_t *
 _hb_directwrite_shaper_font_data_create (hb_font_t *font)
 {
-  return (hb_directwrite_font_data_t *) HB_SHAPER_DATA_SUCCEEDED;
+  IDWriteFontFace *fontFace = (IDWriteFontFace *) (const void *) font->face->data.directwrite;
+
+  /*
+   * Set up variations.
+   */
+  IDWriteFontFace5 *fontFaceVariations = nullptr;
```

Comment:
Did you mean to instead do:
```suggestion
  IDWriteFontFace5 *fontFaceVariations = fontFace;
```

Reply:
Not really. I'm always creating it. And if creation failed, I want to return nullptr.